pkgs/development/web/valum/default.nix

```diff
@@ -1,29 +1,24 @@
-{ stdenv, pkgconfig, fetchFromGitHub, python, glib, vala_0_28, ctpl
+{ stdenv, meson, ninja, pkgconfig, fetchFromGitHub, glib, vala, ctpl
 , libgee, libsoup, fcgi }:
 
 stdenv.mkDerivation rec {
   name = "valum-${version}";
-  version = "0.2.16";
+  version = "0.3.14";
 
   src = fetchFromGitHub {
     owner = "valum-framework";
     repo = "valum";
     rev = "v${version}";
-    sha256 = "0ca067gg5z1798bazwzgg2yd2mbysvk8i2q2v3i8d0d188y2hj84";
+    sha256 = "1w1mipczcfmrrxg369wvrj3wvf76rqn8rrkxbq88aial1bi23kd3";
   };
 
-  nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ python glib vala_0_28 ctpl libgee libsoup fcgi ];
-
-  configurePhase = ''python waf configure --prefix=$out'';
-
-  buildPhase = ''python waf build'';
-
-  installPhase = ''python waf install'';
+  nativeBuildInputs = [ meson ninja pkgconfig ];
+  buildInputs = [ glib vala ctpl libgee libsoup fcgi ];
 
   meta = with stdenv.lib; {
     homepage = https://github.com/valum-framework/valum;
     description = "Web micro-framework written in Vala";
+    license = licenses.lgpl;
```

Comment:
There is no such attribute.

Reply:
Oh, right. Is there a way, I can check metadata locally?

Reply:
Turns out I can just run `nix-build nixos/release.nix`

Reply:
https://github.com/NixOS/nixpkgs/commit/a9d212fea4d6631052d143d7151cacad909beabe#commitcomment-26368327